project-base/storefront/cypress/support/index.ts

```diff
@@ -8,6 +8,8 @@ import { TIDs } from 'tids';
 
 registerCommand({ pollInterval: 500, timeout: 5000 });
 
+const ELEMENTS_WITHOUT_POINTER_EVENTS_FOR_SCREENSHOTS = ['#newsletter-form-privacyPolicy', TIDs.simple_header_contact];
```

Comment:
If I'm being honest, when I take over this PR, I won't know exactly what elements to add to that array 🤔